target/snitch_cluster/README.md

```diff
@@ -1,24 +1,60 @@
-# Snitch cluster target
+# SNAX Cluster Target
 
+## Details
 The Snitch cluster target (`target/snitch_cluster`) is a simple RTL testbench
-around a Snitch cluster. The cluster can be configured using a config file. By default, the config file which will be picked up is `target/snitch_cluster/cfg/default.hsjon`.
+around a SNAX cluster. It is still the same as the Snitch cluster but with SNAX modifications. The cluster can be configured using a config file. By default, the config file which will be picked up is `target/snitch_cluster/cfg/default.hsjon`.
```

Comment:
Does the default one even have snax modifications?

Reply:
Nope. It was the Snitch. We placed all SNAX related to `snax-*.cfg` ones.